liquibase/changes/feat_1181.sql

```diff
@@ -0,0 +1,7 @@
+UPDATE gbfsendpoint
+SET
+  name = 'free_bike_status',
+  is_feature = TRUE
+WHERE
+  name = 'vehicle_status'
+  AND gbfs_version_id::float < 3.0;
```

Comment:
Please remove this condition, the version is a string and it's not guaranteed that it can be converted to a float, for example, v2.1 or 3.1RC
```suggestion
;
```